log.sh

```diff
@@ -0,0 +1,93 @@
+# Top 100
```

Comment:
In log.sh line 1:
# Top 100
^-- SC2148: Tips depend on target shell and yours is unknown. Add a shebang.


In log.sh line 33:
                sum=$((${sum}+${i}))
                       ^-- SC2004: $/${} is unnecessary on arithmetic variables.
                              ^-- SC2004: $/${} is unnecessary on arithmetic variables.


In log.sh line 37:
        for k in ${count[@]}
                 ^-- SC2068: Double quote array expansions, otherwise they're like $* and break on spaces.


In log.sh line 47:
        for i in $(seq 0 $(echo "${#count[@]}-1"|bc))
                         ^-- SC2046: Quote this to prevent word splitting.


In log.sh line 49:
                echo "Response Code: "${code[${i}]}" "
                                      ^-- SC2027: The surrounding quotes actually unquote this. Remove or escape them.
                                      ^-- SC2086: Double quote to prevent globbing and word splitting.


In log.sh line 50:
                echo "Response Count: "${count[${i}]}" "
                                       ^-- SC2027: The surrounding quotes actually unquote this. Remove or escape them.
                                       ^-- SC2086: Double quote to prevent globbing and word splitting.


In log.sh line 51:
                echo "Proportion: "${ratio[${i}]}" %"
                                   ^-- SC2027: The surrounding quotes actually unquote this. Remove or escape them.
                                   ^-- SC2086: Double quote to prevent globbing and word splitting.


In log.sh line 59:
        temp=$(more web_log.tsv | awk -F'\t' '{if(substr($6,1,1)==4)print $5"\t"$6}' > target2.txt)
        ^-- SC2034: temp appears unused. Verify it or export it.


In log.sh line 61:
        codes_count=$(more target2.txt | awk -F'\t' '{print $2}'| sort | uniq -c | awk '{print $1}')
        ^-- SC2034: codes_count appears unused. Verify it or export it.


In log.sh line 71:
                url=$(more target2.txt | awk -F'\t' '{if($2=='$t')print $1}' | sort | uniq -c | sort -nr | head)
                                                              ^-- SC2086: Double quote to prevent globbing and word splitting.


In log.sh line 83:
        echo "----- # Top 100 hosts which visited "$url" # ------"
                                                   ^-- SC2027: The surrounding quotes actually unquote this. Remove or escape them.